Kwf/Cache/Simple.php

```diff
@@ -179,6 +203,11 @@ public static function add($cacheId, $data, $ttl = null)
     {
         if (self::getBackend() == 'memcache') {
             return self::getMemcache()->set(self::_getMemcachePrefix().md5($cacheId), $data, 0, $ttl);
+        } else if (self::getBackend() == 'redis') {
+            if (!$ttl) $ttl = 365*24*60*60; //Set a TTL so it can be evicted http://stackoverflow.com/questions/16370278/how-to-make-redis-choose-lru-eviction-policy-for-only-some-of-the-keys
+            $ret = self::getRedis()->setEx('simple:'.$cacheId, $ttl, serialize($data));
+            if ($ttl) self::getRedis()->expire($cacheId, $ttl);
```

Comment:
Is explicitely setting the expiring date necessary here? The ttl is just use to get a eviction policy for these entries.